pkg/workload/tpcc/worker.go

```diff
@@ -200,12 +212,14 @@ func newWorker(
 }
 
 func (w *worker) run(ctx context.Context) error {
+	rng := rand.New(new(rand.LockedSource))
```

Comment:
could you use `randutil.NewTestRandWithSeed()` instead?

Reply:
Looking at the `NewTestRandWithSeed`, it doesn't look like it's thread safe? The `Rand` object returned seems to just be a regular `rand.New(rand.NewSource(seed))`.

Reply:
ah, i see what you mean. how about `randutil.NewLockedSource()`?

Reply:
Oh yeah that would do it -- although why do we have these proxy functions? They don't seem to provide any additional functionality.

Reply:
for when you don't need to explicitly set a seed (the vast majority of the time) in a test, the utility does it for you https://github.com/msbutler/cockroach/blob/butler-rangefeed-split-event/pkg/util/randutil/rand.go#L77

Reply:
it also has a few helper methods that make things cleaner.

Reply:
Ah I see. Oh and looks like we can't use `randutil.NewLockedSource` -- it creates our custom `lockedSource`, which does not match the package's `LockedSource` and therefore can't be used with `rand.New()`.

Could add more to `randutil`, but looking at the currently available functions, I don't think there's a way to get the same behavior as the current code. `randutil` uses `rand` from `math/rand` whereas the old code in `tpcc` used `rand` from `golang.org/x/exp/rand`. We'll have to pick one and stick with it.

Reply:
Based on some of the commit history from before, it looks like we moved to `golang.org/x/exp/rand` for performance reasons

Reply:
cool, i'm fine using what tpcc has been using.